### PR TITLE
Main Page Setup

### PR DESCRIPTION
**Pull request by teddyrendahl**
First go at the `pcdshub` main site. Site contains:

* Information on how to access the `pcds_env` Conda environment
* Tutorial on how to setup a new miniconda installation and create your own`pcds_env`
* An explanation of our development cycle (fork / pr)
* A list of helpful tutorials and links

### CODE REVIEW

**Comment by n-wbrown** [comment]
You might recommend branch protection in `development.rst`. This can prevent direct pushes to the master fork and PR merges without checks. Git now features a `--rebase` option for the `pull` command but requires a newer version of git than the one we use. Perhaps we should look into upgrading?

File: source/development.rst [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+If you want to make changes to a repository the first step is to create your
+own fork. This allows you to create feature branches without cluttering the
+main repository. It also assures that the main repository is only added to by
+Pull Request and review. Repositories can be forked from the GitHub site like
+`this <https://help.github.com/articles/fork-a-repo>`_. Once this repository is
+created, you can clone into your own workspace.
+
+.. code:: sh
+
+   $ git clone https://github.com/YOUR-USERNAME/REPOSITORY.git
```
Comment (ZLLentz) [inline source/development.rst]: Might suggest doing this as `https://YOUR-USERNAME@github.com/YOUR-USERNAME/REPOSITORY.git`, on some git versions this lets you push/pull from origin without putting your git username in every time

**Comment by teddyrendahl** [comment]
@n-wbrown Where do you think I should mention branch protection? In the development?

**Review by ZLLentz — APPROVED** [review]
I largely just reviewed this for accuracy, I'll come back with my own PR if I think things need to be added later. I added one suggestion here about setting up the local checkout of your fork.

File: source/resources.rst [hunk 2]
```diff
@@ -0,0 +1,27 @@
+=====================
+Recommended Resources
+=====================
+
+Packages We Recommend
+=====================
+* `Pytest <https://docs.pytest.org/en/latest>`_ -- Python Testing Framework
+* `Sphinx <http://www.sphinx-doc.org/en/stable>`_ -- Automated HTML Documentation
+* `Ophyd <http://nsls-ii.github.io/ophyd>`_ -- EPICS Device Abstractions
+* `Bluesky <http://nsls-ii.github.io/bluesky>`_ -- Generalized Experimental Plans
```
Comment (ZLLentz) [inline source/resources.rst]: Also, maybe add flake8 here

File: source/resources.rst [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+External Tutorials and Guides
+=============================
+* `Git Tutorial <http://swcarpentry.github.io/git-novice>`_ -- Thorough Git beginner guide
+* `Numpy Development Guide <https://docs.scipy.org/doc/numpy/dev/gitwash/development_workflow.html>`_ -- A development guide from the folks at NumPy
+* `Git Flight Rules <https://github.com/k88hudson/git-flight-rules>`_ -- Specific instructions for performing advanced tasks in Git
+* `NumPy Documentation Guide <https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt>`_ -- Guide to making NumPy style documentation
+* `Conda <https://conda.io/docs/user-guide/getting-started.html>`_ -- Getting started guide for using Conda
```
Comment (ZLLentz) [inline source/resources.rst]: and maybe link to pep8 style guide here

File: source/development.rst [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+WIP   Commit that is a work in progress
+====  ===
+
+It is also helpful underneath classes and functions to write docstrings. These
+are later converted by Sphinx into HTML documentation. They also are a valuable
+tool for exploration of a codebase within an IPython terminal. Docstrings
+should follow the form described in the `NumPy documentation
+<http://www.sphinx-doc.org/en/stable/ext/example_numpy.html>`_
+
+Merging Changes
```
Comment (n-wbrown) [inline source/development.rst:67]: There isn't a section on starting new repositories so 'Merging Changes' looks like the best section for mentioning it for now.

**Review by n-wbrown — APPROVED** [review]

**Pull request merged**